client/blocks/login/two-factor-authentication/verification-code-form.jsx

```diff
@@ -112,7 +112,7 @@ class VerificationCodeForm extends Component {
 			twoFactorAuthType,
 		} = this.props;
 
-		let helpText = translate( 'Enter the code generated by your Authenticator mobile application.' );
+		let helpText = translate( 'Enter the code generated by your authenticator app.' );
```

Comment:
:disappointed: 17 existing translations will be lost with this change.

Reply:
I'll look if we can manage to keep them despite this message :)